dora/core/server/worker/src/main/java/alluxio/worker/http/HttpServerHandler.java

```diff
@@ -95,6 +97,89 @@ public void channelRead0(ChannelHandlerContext ctx, HttpObject msg) {
     }
   }
 
+  private Map<String, String> parseRequestParameters(String requestUri) {
+    requestUri = requestUri.substring(requestUri.indexOf("?") + 1);
+    String[] params = requestUri.split("&");
+    Map<String, String> parametersMap = new HashMap<>();
+    for (String param : params) {
+      String[] keyValue = param.split("=");
+      parametersMap.put(keyValue[0], keyValue[1]);
+    }
+    return parametersMap;
+  }
+
+  private FullHttpResponse dispatch(Channel channel, HttpRequest httpRequest) {
+    String requestUri = httpRequest.uri();
+    // parse the request uri to get the parameters
+    String requestMapping = getRequestMapping(requestUri);
+    Map<String, String> parametersMap = parseRequestParameters(requestUri);
+
+    // parse the URI and dispatch it to different methods
+    switch (requestMapping) {
+      case "page":
+        return doGetPage(parametersMap, channel, httpRequest);
+      case "files":
```

Comment:
@JiamingMai can it be "list" or other keywords since the directory may contain all subdir?